.travis.yml

```diff
@@ -1,5 +1,25 @@
 language: java
 
-jdk:
-  - oraclejdk8
-  - oraclejdk9
+install: true
+
+before_install:
+  - chmod +x gradlew
+
+stages:
+  - name: build
+  - name: snapshot
+    if: branch = develop
+  - name: release
+    if: branch = master
+
+jobs:
+  include:
+    - stage: build
+      jdk: oraclejdk8
+      script: ./gradlew build
+    - jdk: oraclejdk9
```

Comment:
The hyphen here looks odd to me - is it really in the right place? Or should it be indented more?